arches/app/templates/javascript.htm

```diff
@@ -189,6 +189,7 @@
         select-a-nodegroup='{% trans "Select a nodegroup" as selectANodegroup %} "{{ selectANodegroup|escapejs }}"'
         no-relationships-added='{% trans "No Relationships Added" as noRelationshipsAdded %} "{{ noRelationshipsAdded|escapejs }}"'
         relate-resource='{% trans "Relate Resource" as relateResource %} "{{ relateResource|escapejs }}"'
+        configure-related-resource-relationship='{% trans "Configure Related Resource Relationship" as configureRelateResourceRelationship %} "{{ configureRelateResourceRelationship|escapejs }}"'
```

Comment:
Just a heads up: the left side is `related` and the right side is `relate`: would be great to have both the same

Reply:
Whoops that is definitely a typo